util/netcdf_generator.py

```diff
@@ -86,9 +86,13 @@ def _filter_params(self, ds, params):
             params_to_filter.append('annotations')
 
         for key in ds.data_vars:
-            if key not in params_to_filter:
+            # do not remove qc parameters '%s_qc_executed' and '%s_qc_results'
+            if key not in params_to_filter and not self._is_qc_parameter(key):
```

Comment:
params_to_filter is badly named (it's not something you did here), but if you can please rename it so that it's more clear.  when I see filter I think remove, but params_to_filter does not contain a list of parameters to remove, rather it contains a list of parameters to keep.       please rename, suggest: params_to_keep.